docs/sandbox.md

```diff
@@ -1,90 +0,0 @@
-# Getting Started with Scalar DL Sandbox
```

Comment:
Not fully sure, but is this file also used for generating [this page](https://scalardl.readthedocs.io/en/latest/sandbox/) based on `mkdocs.yml`?

Reply:
Oh, Yes! Thank you! Fixed in [f261bc6](https://github.com/scalar-labs/scalardl/pull/25/commits/f261bc687b56ab118c8b90ff1b5182c7ca44eb45).